ocaml/xapi/xapi.ml

```diff
@@ -167,8 +167,18 @@ let register_callback_fns() =
     Locking_helpers.Thread_state.acquired (Locking_helpers.Process("stunnel", pid)) in
   let unset_stunnelpid task_opt pid =
     Locking_helpers.Thread_state.released (Locking_helpers.Process("stunnel", pid)) in
+  let stunnel_destination_is_ok addr =
+    Server_helpers.exec_with_new_task "check_stunnel_destination" (fun __context ->
+      let hosts = Db.Host.get_refs_where ~__context ~expr:(
+        Eq (Field "address", Literal addr)
```

Comment:
I always forget to ask, is this query language documented somewhere?

Reply:
No, not as far as I'm aware.

Reply:
:(